daemon/logger/ring.go

```diff
@@ -0,0 +1,210 @@
+package logger
+
+import (
+	"errors"
+	"sync"
+	"sync/atomic"
+
+	"github.com/Sirupsen/logrus"
+)
+
+const (
+	defaultRingMaxSize = 1000000 // 1MB
```

Comment:
It's kinda hard to count zeroes. I think 1024 * 1024 would be simpler to read. Also more scientific :microscope: